debian/pam-configs/authd.in

```diff
@@ -1,6 +1,6 @@
 Name: Authd authentication
 Default: yes
-Priority: 280
+Priority: 1050
```

Comment:
Did you pick this random number or is there any strategy? :)

Reply:
Same curiosity here :-D

Reply:
Oops, I forgot to add a comment here to explain the number.

The reasoning is basically: pam_pwquality uses priority 1024, so I wanted us to come before it but not by too much to not override other things that could be configured in the user's system.

Reply:
No comment was added yet, right?

Reply:
Done now